src/main/java/cubicchunks/worldgen/generator/custom/structure/CubicCaveGenerator.java

```diff
@@ -66,7 +66,7 @@
      * Vanilla value: 7 Multiply by 16 and divide by 8: there are 16 cubes in a vanilla chunk, but only one cube per 8
      * has caves generated
      */
-    private static final int CAVE_RARITY = 16 * 7 / (2 * 2 * 2);
+    private static final int CAVE_RARITY = 16 * 7 / (2 * 2 * 2); // TODO: 32³ Cubes
```

Comment:
Caves generation is already partially 32x32x32 and shouldn't be too hard to make work. But you would need whole 32x32x32 worldgen to go with it.